src/main/kotlin/io/openfuture/state/service/DefaultWalletService.kt

```diff
@@ -20,4 +22,25 @@ class DefaultWalletService(private val walletRepository: WalletRepository) : Wal
                 ?: throw NotFoundException("Wallet not found")
     }
 
+    override suspend fun addTransactions(requests: Set<TransactionRequest>) {
+        requests.forEach {
+            val wallet = findByAddress(it.walletAddress)
+            val transaction = Transaction(
```

Comment:
why you have to specify properties names?
blockchainType = it.blockChainType,
hash = it.hash, etc?

Why you cannot just have a proper properties order to have something like Transaction(it.blockchain, it.hash, ...)?